package.json

```diff
@@ -21,12 +29,21 @@
 		"url": "https://github.com/mathiasbynens/String.prototype.includes.git"
 	},
 	"bugs": "https://github.com/mathiasbynens/String.prototype.includes/issues",
-	"files": [
-		"LICENSE-MIT.txt",
-		"includes.js"
-	],
 	"scripts": {
-		"test": "node tests/tests.js",
-		"cover": "istanbul cover --report html --verbose --dir coverage tests/tests.js"
+		"pretest": "es-shim-api --bound",
```

Comment:
I should remove `--bound`, right?

Reply:
no, at the moment, it requires it because the index isn't `===` to the polyfill.